pkg/virtctl/vnc/vnc.go

```diff
@@ -97,22 +106,35 @@ func (o *VNC) Run(cmd *cobra.Command, args []string) error {
 
 	// execute remote viewer
 	go func() {
-		cmnd := exec.Command("remote-viewer", fmt.Sprintf("vnc://127.0.0.1:%d", port))
-		err := cmnd.Run()
+		args := []string{fmt.Sprintf("vnc://127.0.0.1:%d", port)}
+		if glog.V(4) {
+			args = append(args, "--debug")
+			glog.Infof("remote-viewer commandline: %v", args)
+		}
+
+		cmnd := exec.Command("remote-viewer", args...)
+		output, err := cmnd.CombinedOutput()
 		if err != nil {
 			glog.Errorf("remote-viewer execution encountered an error: %v", err)
 		}
+		if err != nil || glog.V(2) {
```

Comment:
I would prefer Infof and the easier to read form:

```
if err != nil {
  glog.V(2).Infof(...)
}
```

Reply:
Oh and it should be `if err ==nil`.

Reply:
Hm, what I had in mind is something like this:

```golang
if err != nil {
  glog.Errorf("remote-viewer execution failed: %v, output: %v", err, output)
} else {
  glog.V(2).("remote-viewer output: %s", output)
}

Reply:
done